app/lib/methods/subscriptions/rooms.ts

```diff
@@ -153,7 +170,7 @@ const createOrUpdateSubscription = async (subscription, room) => {
 			Encryption.decryptPendingMessages(tmp.rid);
 		}
 
-		const batch = [];
+		const batch = [] as TSubscriptionModel[] & TMessageModel[];
```

Comment:
I mentioned this one on our group the other day
```suggestion
		const batch: Model[] = [];
```